openhcl/underhill_core/src/worker.rs

```diff
@@ -1228,8 +1228,14 @@ async fn new_underhill_vm(
     // The amount of memory required by the GET igvm_attest request
     let attestation = get_protocol::IGVM_ATTEST_MSG_MAX_SHARED_GPA as u64 * hvdef::HV_PAGE_SIZE;
 
-    // TODO: determine actual memory usage by NVME/MANA. hardcode as 10MB
-    let device_dma = 10 * 1024 * 1024;
+    // TODO: retrieve this via the host; this heuristic is temporary.
```

Comment:
A similar hint already exist for private pool https://microsoft.visualstudio.com/OS/_git/os.2020?path=/onecore/vm/worker/wpcore/WorkerFileLoader.cpp&version=GBofficial/rs_base2_hyp&line=400&lineEnd=400&lineStartColumn=17&lineEndColumn=89&lineStyle=plain&_a=contents

FYI if we want piggyback on it

Reply:
I think this is the right hint.  I think we want to put this under a different DT node than "servicing", given that it is more generally required

Reply:
There is pending PR which does that: https://microsoft.visualstudio.com/OS/_git/os.2020/pullrequest/11986480